kuksa-persistence-provider/src/storage.rs

```diff
@@ -0,0 +1,31 @@
+/********************************************************************************
+* Copyright (c) 2024 Contributors to the Eclipse Foundation
+*
+* This program and the accompanying materials are made available under the
+* terms of the Apache License 2.0 which is available at
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* SPDX-License-Identifier: Apache-2.0
+********************************************************************************/
+
+pub mod filestorage;
```

Comment:
why split storage and filestorage into two files? Just move this to filestorage imo